### ISSUE TITLE
*mlocate* should be denpendency

### ISSUE THREAD

**Issue by cinqs** [tensorflow/tensorflow#12168]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
  docker centos 7
- **TensorFlow installed from (source or binary)**:
  source
- **TensorFlow version (use command below)**:
  r1.2
- **Python version**: 
  2.7.5
- **Bazel version (if compiling from source)**:
  0.5.3
- **CUDA/cuDNN version**:
  none
- **GPU model and memory**:
  none
- **Exact command to reproduce**:

    ./configure

### Describe the problem

    `mlocate` command should be *claimed* as a dependency in your installation instruction

**Comment by gunan** [comment]
It looks to be added with this pull request.
https://github.com/tensorflow/tensorflow/pull/9641

@vrv @andydavis1 @zhangyaobit Could you comment on our dependence on mlocate?
If it is a hard dependency, I agree that we should document it.

**Comment by andydavis1** [comment]
@vivek-rane Can you comment on "mlocate"?

**Comment by vivek-rane** [comment]
We don't need mlocate any more. The configure and build scripts have changed since that commit to not use mlocate.

**Comment by gunan** [comment]
Thanks for the clarification @vivek-rane 
@cinqs is there a reason you believe we need `mlocate` as a dependency?

**Comment by cinqs** [comment]
@gunan , Hi, I don't have specific reason why `mlocate` is a must. I filed this issue only because when I compiling your source, I ran across the situation that `mlocate` is a must of dependency.

Since I was compiling from `r1.2`, which in fact is the version your official API document supporting then. 
And, I was compiling the source inside a docker container. The image is `CentOS 7`

In conclusion, no particular reason. And sorry for the late response, was quite a busy summer.
